tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/hive/HiveTableManager.java

```diff
@@ -52,22 +50,50 @@
     private final QueryExecutor queryExecutor;
     private final HdfsDataSourceWriter hdfsDataSourceWriter;
     private final String testDataBasePath;
+    private final HiveThriftClient hiveThriftClient;
     private final String databaseName;
     private final String hiveDatabasePath;
     private final boolean analyzeImmutableTables;
     private final boolean analyzeMutableTables;
 
     @Inject
-    public HiveTableManager(QueryExecutor queryExecutor,
+    public HiveTableManager(
+            QueryExecutor queryExecutor,
             HdfsDataSourceWriter hdfsDataSourceWriter,
             TableNameGenerator tableNameGenerator,
             @Named("tests.hdfs.path") String testDataBasePath,
             @Named("databaseName") String databaseName,
             @Named("path") String databasePath,
             @Named("analyze_immutable_tables") boolean analyzeImmutableTables,
-            @Named("analyze_mutable_tables") boolean analyzeMutableTables)
```

Comment:
change parameter name to `inject_stats_for_immutable_tables` and `inject_stats_for_mutable_tables`. Or just single `inject_stats`. 
I presumed that we want to keep the `ANALYZE` flow. Then the parameters would be valid for the old functionality, but the new one would still need new params. If we are scapping old code, scrap the old parameters too.